sdk/javascript/scripts/ci/test.sh

```diff
@@ -2,5 +2,5 @@
 
 set -ex
 
-TEST_MODE=$([ "$1" = "code-coverage" ] && echo "coverage:test" || echo "test")
-npm run "${TEST_MODE}"
+TEST_MODE=$([ "$1" = "code-coverage" ] && echo "test:jenkins" || echo "test")
+TEST_SCRIPT='test' npm run "${TEST_MODE}"
```

Comment:
this isn't running anything is it?

Reply:
Instead of having two scripts for code coverage(test and vectors), I created a env variable which state which to run.  It should run either test or vectors.
I can move back to two scripts if that is more readable.

Reply:
i think two would be more readable